### PR TITLE
Add trustee candidate details

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

File: pages/ukpa/trustees.md [hunk 1]
```diff
@@ -18,12 +18,79 @@ To be eligible as a trustee you must be at least 16 years of age and not otherwi
...
+to make the most of the opportunities available to a registered charity, to broaden its scope beyond the annual conference,
+to be utterly transparent and accountable to its members and to be a driving force of the UK Python community. 
+
+I have 15 years' experience as a charity trustee within the UK Scout Association and am currently a trustee of Mersey 
+Weaver Scout District (charity No. 1145482).
+
+#### Kristian Glass
+<img class="avatar" src="https://www.gravatar.com/avatar/403476fc38e4e0be93bf03282a33e72c/?s=120" alt="Kristian Glass" />
+
+'m Kristian Glass; I was treasurer for PyCon UK 2016 and 2017, and set up and have run the PyCon UK Society Ltd company
```
Comment (alexwlchan) [inline pages/ukpa/trustees.md]: Missing a leading “I” – copy/paste error?

File: pages/ukpa/trustees.md [hunk 2]
```diff
@@ -18,12 +18,79 @@ To be eligible as a trustee you must be at least 16 years of age and not otherwi
...
+to back it. I am also the director of Moorhen Solutions Ltd (my own consultancy company) and ran NHS Hack Day Cambridge
+(a not-for-profit "hackathon"), as well as other having other experience in the not-for-profit, events and financial
+spaces I'm glad to see the UKPA created, and want to contribute my experience towards the success of both it and of PyCon UK.
+
+#### Peter Inglesby
+<img class="avatar" src="https://www.gravatar.com/avatar/510f52d9d1b77f5088505ed49608f5d7/?s=120" alt="Peter Inglesby" />
+
+I have been involved in running PyCon UK since 2013, and chaired the conference in 2016 and 2017. I will be standing down
+as conference chair after this year's conference, but I would love to continue to work for the UK Python community by 
+serving as a trustee of the UKPA. There are lots of things that the UKPA could be and do! If elected, than in its first
```
Comment (alexwlchan) [inline pages/ukpa/trustees.md]: “*then* in its first year”?

File: pages/ukpa/trustees.md [hunk 3]
```diff
@@ -18,12 +18,79 @@ To be eligible as a trustee you must be at least 16 years of age and not otherwi
...
+to support the conference committee and reduce the burden on our volunteers.
+
+#### Chloe Parkes
+This year (2017) will be my 3rd year attending PyCon UK. The first year I attended helped me through a lot, including confidence
+issues. That same year I decided I wanted to be a committee member, and help organise a conference I personally found life/career
+changing. I'm so proud to have been part of the conference committee so far, playing a part in getting it all to run smoothly,
+and I hope to be doing it for a good while longer. I would really enjoy being able to support the UKPA in anyway that I can.
+
+#### Nicholas Tollervey
+<img class="avatar" src="https://www.gravatar.com/avatar/e8d4e155929363bd22a7852494d18c25/?s=120" alt="Peter Inglesby" />
```
Comment (alexwlchan) [inline pages/ukpa/trustees.md]: Alt text is wrong.

**Review by alexwlchan — APPROVED** [review]
A couple of minor tweaks, but otherwise LGTM. 👍

File: pages/ukpa/trustees.md (hunk 3 above)
Comment (meatballs) [reply pages/ukpa/trustees.md]: Thank you. All fixed now.

**Pull request merged**